Source/Scene/GlobeSurfaceShaderSet.js

```diff
@@ -187,6 +191,12 @@ define([
             vs.sources.push(getPositionMode(sceneMode));
             vs.sources.push(get2DYPositionFraction(useWebMercatorProjection));
 
+            if (receiveShadows) {
+                // TODO : should have a better way of modifying the GlobeVS and GlobeFS shaders...
+                vs.sources[1] = ShadowMapShader.createReceiveShadowsVertexShader(vs.sources[1]);
+                fs.sources[0] = ShadowMapShader.createReceiveShadowsFragmentShader(fs.sources[0], frameState.context);
+            }
```

Comment:
This was the easiest way for me to modify the Globe shader to support shadows. I am open to other ideas.